.github/workflows/main.yml

```diff
@@ -0,0 +1,40 @@
+name: Testing Interoperability
+run-name: ${{ github.actor }} is testing out GitHub Actions 🚀
+on: workflow_dispatch
+jobs:
+  Testing_Interoperability:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Downloads assets
+        uses: robinraju/release-downloader@v1.7
+        with:
+          latest: true
+          fileName: "*"
+      - name: Unzip
+        uses: montudor/action-zip@v1
+        with:
+          args: unzip -qq *.zip -d executables
+      - name: Run Interoperability script
+        run: |
+          python3 -m venv .venv
+          source .venv/bin/activate
+          pip install -r requirements.txt
+          cd executables
+          for i in ./* ; do for j in ./*; do python3 ./../interoperability_report.py -P $i -S $j -o=./../general_report.xml; done; done
+      - name: XUnit Viewer
+        id: xunit-viewer
+        uses: AutoModality/action-xunit-viewer@v1
+        with:
+          results: ./general_report.xml
```

Comment:
can we call this file `junin_interoperability_report.xml` ?

`general_report.xml` is... quite general 😅

Reply:
junin_interoperability_report.xml or junit_interoperability_report.xml ?

Reply:
i changed it to junit_interoperability_report.xml

Reply:
yes, `junit`, I had a typo there